ci/build/build-pipeline.yaml

```diff
@@ -256,31 +256,43 @@ spec:
       plan:
       - get: cloudhsm-config
         trigger: true
-      - task: build-cloudhsm
-        privileged: true
-        config:
-          <<: *build_config
-          params:
-            CONTEXT: cloudhsm-config/cloudhsm
-          inputs:
-          - name: cloudhsm-config
-      - put: cloudhsm-client-image
-        params:
-          <<: *image_put_params
-          additional_tags: cloudhsm-config/.git/short_ref
-
-      - task: build-cloudhsm-jce
-        privileged: true
-        config:
-          <<: *build_config
-          params:
-            CONTEXT: cloudhsm-config/cloudhsm/jdk-jce-image
-          inputs:
-          - name: cloudhsm-config
-      - put: cloudhsm-jce-image
-        params:
-          <<: *image_put_params
-          additional_tags: cloudhsm-config/.git/short_ref
+      - in_parallel:
+          limit: 3
```

Comment:
You can probably remove these limits now... It was only there previous to reduce push errors to harbor (and there's only two do blocks)